client/src/components/ProjectSamplesTable.js

```diff
@@ -247,13 +246,15 @@ export const ProjectSamplesTable = ({
           />
         </Box>
         <Box>
-          <DownloadModal
-            disabled={!isMetadataOnlyAvailable || hasFilter}
-            label="Download Sample Metadata"
-            icon={<DownloadIcon color="brand" />}
-            resource={project}
-            publicComputedFile={metadataComputedFile}
-          />
+          {metadataComputedFile && (
+            <DownloadModal
+              label="Download Sample Metadata"
+              icon={<DownloadIcon color="brand" />}
+              resource={project}
+              publicComputedFile={metadataComputedFile}
+              disabled={!isMetadataOnlyAvailable || hasFilter}
+            />
+          )}
```

Comment:
The same comment here. We should still render the button but make it disabled here.